andela/baby-face-cfh
# #153742463 user sends invites to 11 players

#### What does this PR do?
- User should be able to search for a player.
- User should be able to send an invite to a player for a game

#### Description of Task to be completed?
- Create a search controller and route
- Create an invite controller and route
- Add a view in the template for users to be able to search and invite player

#### How should this be manually tested?
This can be tested when you go to the game page, you will see the invite player button.

#### Any background context you want to provide?
NIll

#### What are the relevant pivotal tracker stories?
#153742463

#### Screenshots (if appropriate)
<img width="1256" alt="screen shot 2018-01-09 at 12 05 23 pm" src="https://user-images.githubusercontent.com/12601490/34718042-81816bfa-f535-11e7-889b-f31414c8c8de.png">


#### Questions:
NIll
